static/js/components/SurveyLinkList.jsx

```diff
@@ -211,7 +211,7 @@ const SurveyLinkList = ({ ra, dec, id }) => {
       />
       <SurveyLink
         name="SDSS"
-        url={`http://skyserver.sdss.org/dr16/en/tools/chart/navi.aspx?opt=G&ra=${ra}&dec=${dec}&scale=0.25`}
+        url={`https://skyserver.sdss.org/dr16/en/tools/chart/navi.aspx?opt=G&ra=${ra}&dec=${dec}&scale=0.25`}
```

Comment:
This link,

https://skyserver.sdss3.org/public/en/tools/chart/navi.aspx?opt=G&ra=0&dec=0&scale=0.1981

Gives 

<img width="995" alt="Screen Shot 2020-11-27 at 3 45 42 PM" src="https://user-images.githubusercontent.com/2769632/100489190-ac2a2380-30c7-11eb-9418-70c6ede8872c.png">